plugin.xml

```diff
@@ -1,73 +1,58 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:amazon="http://schemas.android.com/apk/lib/com.amazon.device.ads" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-push" version="1.10.4">
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:amazon="http://schemas.android.com/apk/lib/com.amazon.device.ads" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-push" version="2.0.0-rc3">
   <name>PushPlugin</name>
   <description>
-		This plugin allows your application to receive push notifications on Android, iOS and Windows devices.
-    Android uses Google Cloud Messaging.
+    This plugin allows your application to receive push notifications on Android, iOS and Windows devices.
+    Android uses Firebase Cloud Messaging.
     iOS uses Apple APNS Notifications.
     Windows uses Microsoft WNS Notifications.
-	</description>
+  </description>
   <license>MIT</license>
   <js-module src="www/push.js" name="PushNotification">
     <clobbers target="PushNotification"/>
   </js-module>
   <engines>
     <engine name="cordova" version=">=6.4.0"/>
-    <engine name="cordova-android" version=">=6.0.0"/>
-    <engine name="cordova-ios" version=">=4.3.0"/>
+    <engine name="cordova-android" version=">=6.2.1"/>
+    <engine name="cordova-ios" version=">=4.4.0"/>
   </engines>
-  <preference name="SENDER_ID" default="85075801930"/>
   <platform name="android">
+    <hook type="before_plugin_install" src="scripts/copyAndroidFile.js"/>
```

Comment:
Yeah, it's still needed as the classpath "com.google.gms:google-services:3.0.0" has to end up in the buildscript dependencies section. This will screw us up for PGB.